app/views/two_factor_authentications/_confirm.haml

```diff
@@ -0,0 +1,22 @@
+.row
+  .col-md-12
+    %h3 Confirm two factor authentication
+    %p Two factor authentication is not fully activated yet
+
+    .row
+      .col-md-6
+        %p Please scan the QR code with a TOTP capable app (eg Google authenticator).
```

Comment:
I'd love to not encourage Google usage :stuck_out_tongue: So at least list other apps. I'm not familiar with TOTP, I think [FreeOTP](https://freeotp.github.io/) is one. If someone can check if it is available on F-droid? Or an other trusted app listed there. That would be awesome!

Reply:
Yeah, totally. If you'd recommend FreeOTP then I can write that. Do you know by chance if there is also an alternative app for iOS?

Reply:
> Do you know by chance if there is also an alternative app for iOS?

Yes, FreeOTP!